src/config/key-redirects.json

```diff
@@ -16,7 +16,7 @@
     "restrictionAddressDeletions": "account",
     "accountAliasName": "account",
     "cosignatories": "account",
-    "multisigAccounts": "account",
+    "multisigAccounts_": "account",
```

Comment:
it's typo for in the end of the word `_` ?

Reply:
It is item of. `multisigAccounts` array. I agree this is not intuitive. I will make some changes